client/src/Poem.jsx

```diff
@@ -10,11 +12,16 @@ PoemLine.propTypes = {
   line: PropTypes.string,
 };
 
+const NBSP = '\u00A0'; // Unicode value for non-breaking space
+
+PoemLine.defaultProps = {
+  line: NBSP,
+};
+
 const Poem = ({ poem }) => {
-  const poemLineNodes = poem.text.split(/\n/).map((line, index) => {
-    const NBSP = '\u00A0'; // Unicode value for non-breaking space
-    return <PoemLine line={(line.trim().length === 0) ? NBSP : line} key={index} />;
-  });
+  const poemLineNodes = poem.text.split(/\n/).map(line =>
+    <PoemLine line={(line.trim().length === 0) ? NBSP : line} key={_.uniqueId()} />,
```

Comment:
I'm fine with this rule in principle. I'm not a fan of its application here, though, because this array is effectively immutable -- the order isn't going to change. Ultimately, I think we're going to be storing poem lines as entities in their own right, so they will have database IDs that we can use. For now, I'm hesitant to bring in underscore for this. Looking into whether the rule has any flexibility we can configure.